snmp/tests/test_profiles.py

```diff
@@ -1274,9 +1274,9 @@ def test_palo_alto(aggregator):
 
 
 @pytest.mark.usefixtures("dd_environment")
-def test_cisco_asa_5525(aggregator):
-    profile = "cisco-asa-5525"
-    run_profile_check(profile)
+def test_cisco_asa(aggregator):
+    profile = "cisco-asa"
+    run_profile_check('cisco_asa', profile)
```

Comment:
Could you rename cisco_asa  to cisco-asa for the community string / snmprec file ?

Mainly for consistency, see also: https://github.com/DataDog/integrations-core/pull/7914

Reply:
+1